systers_portal/meetup/views.py

```diff
@@ -1457,3 +1459,19 @@ def get(self, request, *args, **kwargs):
             # TODO: configure logging and log the unknown status
         return super(CancelMeetupLocationJoinRequestView, self).get(request, *args,
                                                                     **kwargs)
+
+
+@api_view(['POST'])
+def api_for_vms(request):
+    # API to communicate with VMS
+    date = request.data['date']
+    meetups = Meetup.objects.filter(date__gte=date)
+    # fetching all meetups whose start date is greater than or equal to the date posted
+    meetup_list = list()
+    for meetup in meetups:
+            meetup_data = {}
+            meetup_data['event_name'] = meetup.title
+            meetup_data['start_date'] = meetup.date
+            meetup_data['venue'] = meetup.venue
+            meetup_list.append(meetup_data)
+    return JsonResponse(meetup_list, safe=False)
```

Comment:
does the JSON response have events arranged in increasing order of date. It not add a filter to sort your queryobject.